package.json

```diff
@@ -447,9 +460,9 @@
                 "default": null
               },
               "runspaceId": {
-                "type": "number",
+                "type": "string",
                 "description": "Optional: The ID of the runspace to debug in the attached process.  Defaults to 1.  Works only on PowerShell 5 and above.",
-                "default": 1
+                "default": "1"
```

Comment:
One of the things I did in #1775 was made the default for `processId` to be null and then prompt for it when resolving the debug config:

https://github.com/PowerShell/vscode-powershell/blob/1db228398cadea0388995a2d0351a69dd2fe4bd3/src/features/DebugSession.ts#L81-L83

@rkeithhill @rjmholt if we want to make prompting for PickRunspace to be the default, it should probably be implemented in a similar fashion.